src/ophyd_async/epics/_backend/_p4p.py

```diff
@@ -238,9 +238,8 @@ def __init__(self, datatype: Optional[Type[T]], read_pv: str, write_pv: str):
         self.converter: PvaConverter = DisconnectedPvaConverter()
         self.subscription: Optional[Subscription] = None
 
-    @property
-    def source(self, name: str):
-        return f"pva://{self.read_pv}"
+    def source(self, name: str = "") -> str:
+        return f"ca://{self.read_pv}"
```

Comment:
How is this property -> method API change in the p4p backend related to signal logging?
Why are p4p sources being marked as ca?

Reply:
Ah thanks for spotting that, the 'ca' was a typo.